package DualStick;

import "box2d.proto";


message PbDualStick
{
    required box2d.PbVec2 pbv2Move = 1;
    required box2d.PbVec2 pbv2Shoot = 2;
}
